register/models.py

```diff
@@ -122,8 +139,14 @@ def send_reimbursement(self, request):
         self.save()
 
     def confirm(self, cancellation_url):
+        if self.status in [APP_ACCEPTED,APP_PENDING,APP_REJECTED]:
+            raise Http404('Unfortunately this invite doesn\'t even exist')
```

Comment:
I don't get this. If the application is accepted, pending or rejected, then doesn't exist?